package/kodi/Config.in

```diff
@@ -103,6 +106,7 @@ menuconfig BR2_PACKAGE_KODI
 	select BR2_PACKAGE_SQLITE
 	select BR2_PACKAGE_TAGLIB
 	select BR2_PACKAGE_TINYXML
+	select BR2_PACKAGE_TINYXML2
```

Comment:
Are you sure both are needed here?

Reply:
not all plugins have moved to tinyxml2 yet, so yes we need both.

Reply:
i see, thanks for the clarification